docs/table/mixin_columns.rst

```diff
@@ -238,10 +238,19 @@ that contain mixin columns:
 
 **ASCII table writing**
 
-Mixin columns can be written out to file using the `astropy.io.ascii` module,
+Tables with mixin columns can be written out to file using the `astropy.io.ascii` module,
 but the fast C-based writers are not available.  Instead the legacy pure-Python
```

Comment:
I recall some discussion of not calling the pure-Python version legacy? Or was that just the opposite, to call it legacy? Sadly I couldn't find the issue now.
The former would make more sense to me, as the fast writer doesn't provide all the functionality.

Reply:
Good catch, the pure-Python writers are definitely not legacy.